api/app/signals/apps/reporting/csv/utils.py

```diff
@@ -14,6 +16,23 @@
 logger = logging.getLogger(__name__)
 
 
+def zip_csv_files(using: str) -> None:
+    """
+    Writes zip file of the generated csv files in the {now:%Y}/{now:%m}/{now:%d} folder
+
+    :returns None:
+    """
+    storage = _get_storage_backend(using=using)
+    now = timezone.now()
+    src_folder = f'{storage.location}/{now:%Y}/{now:%m}/{now:%d}'
```

Comment:
What will happen if we decide to do several CSV dumps on 1 day? Do we want to get 1 big zip file containing all CSV files OR 1 zip file per run containing only the CVS files created in that run?

Reply:
Currently you will get duplicate csv files within the zip. But the this zip task is scheduled once a day.
If we could force the csv files to have the same prefix (timestamp) we could zip files per 'session'. Currently files in 1 session can have different timestamp. I did try to match them by prefix but that did not work out. Other approach is return the generated file name, and accumulate them

Reply:
I implemented the 2nd option, all the save csv will now return a list of the created files. This list is used for zipping the result. Multiple dumps will only include csv of current 'session'